### PR TITLE
[DOPS-1364] add CI

### PR DESCRIPTION
**Pull request by Cre-eD**
# Task

[DOPS-1364]: add CI process.

## Changes

1. Add CI process for parachain image.
2. Add CI for relaychain image.
3. Add CI for parachain environment image.
4. Add Chainspec and fix CI.

## Author

Signed-off-by: Dmitriy Creed <creed@soramitsu.co.jp>

[DOPS-1364]: https://soramitsu.atlassian.net/browse/DOPS-1364

### CODE REVIEW

File: Jenkinsfile [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+    }
+    agent {
+        label agentLabel
+    }
+    stages {
+        stage('Secret scanner'){
+            steps {
+                script {
+                    gitNotify('main-CI', 'PENDING', 'This commit is being built')
+                    docker.withRegistry( 'https://' + registry, dockerBuildToolsUserId) {
```
Comment (BulatSaif) [inline Jenkinsfile:27]: no need for docker, secret scanner uses it own docker to run the test.

**Review by BulatSaif — APPROVED** [review]

**Review by BulatSaif — APPROVED** [review]

**Review by ravenolf — APPROVED** [review]
LGTM

**Review by Mateomoon — APPROVED** [review]
TLDR
😁
